packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -27,15 +28,16 @@ export abstract class InversifyHttpAdapter<
   TResponse,
   TNextFunction extends (err?: unknown) => void,
 > {
+  protected readonly corsOptions: CorsOptions;
```

Comment:
this adapter is agnostic, but it's relying on `cors` which is coupled to express. What would be the approach in hono / uwebsockets?